### ISSUE TITLE
Folder not found issue in Course 2 - Part 2 - Lesson 3

### ISSUE THREAD

**Issue by amitness** [lmoroney/dlaicourse#11]
This [notebook](https://github.com/lmoroney/dlaicourse/blob/master/Course%202%20-%20Part%202%20-%20Lesson%203%20-%20Notebook.ipynb) needs to have its code updated.

I get this error.
![image](https://user-images.githubusercontent.com/8587189/54072041-6dbe5680-429d-11e9-85f2-7e06fea1e93e.png)

On inspection, I found that the folders are actually named horses and humans.
![image](https://user-images.githubusercontent.com/8587189/54072050-7b73dc00-429d-11e9-81fc-0f70dab2b7a6.png)

So, please update the folder names in this part of code.
![image](https://user-images.githubusercontent.com/8587189/54072057-8e86ac00-429d-11e9-9339-dbbacfd113c9.png)

**Comment by slabua** [comment]
Yes, the correct paths for the validation folders should be set as follows:
```
# Directory with our training horse pictures
validation_horse_dir = os.path.join('/tmp/validation-horse-or-human/horses')

# Directory with our training human pictures
validation_human_dir = os.path.join('/tmp/validation-horse-or-human/humans')
```

**Comment by slabua** [comment]
Also the following notebook is affected by the same issue:
[Course 1 - Part 8 - Lesson 4 - Notebook.ipynb](https://github.com/lmoroney/dlaicourse/blob/master/Course%201%20-%20Part%208%20-%20Lesson%204%20-%20Notebook.ipyn)

**Comment by ryandawsonuk** [comment]
It seems a fix has beeen submitted in https://github.com/lmoroney/dlaicourse/pull/15
